tests/ms_azure/test_utils.py

```diff
@@ -379,7 +379,7 @@ def test_update_existing_skus_x86_gen1_default(
         assert res == [
             VMISku.from_json(x)
             for x in [
-                {"imageType": "x64Gen1", "skuId": "plan1", "securityType": ["trusted"]},
+                {"imageType": "x64Gen1", "skuId": "plan1"},
                 {"imageType": "x64Gen2", "skuId": "plan1-gen2", "securityType": ["trusted"]},
```

Comment:
Could we cover `arm64Gen2` in the unit test?

Reply:
Sure!

Reply:
@lslebodn I've added a whole new test named `test_update_existing_skus_mixed_arches` which covers the case we're interested in, please let me know WDYT